core/session.ts

```diff
@@ -120,6 +128,8 @@ export class Session {
       return this.test(actionOptions.sqlxConfig.name).dataset(actionOptions.sqlxConfig.dataset);
     }
 
+    let schema = actionOptions.sqlxConfig.schema;
```

Comment:
for some reason i can't respond to the existing comment thread on this, github not being helpful...

anyways: thanks a lot for finding this bug! would you mind merging in https://github.com/dataform-co/dataform/pull/324 to fix? it includes a unit test for the fix too.

Reply:
Don't worry about this one any more - I wanted to get that fix in, so have merged it, if you pull from master (and revert your changes to this file) everything should be good now.